### PR TITLE
Update Readme

### LINKED ISSUE #227: Update documentation in analytics-reporter-api repository README.md
## Background

Updates to `analytic-reporter-api` might require updates to the README.md file

## Acceptance Criteria

- [x] Any required changes that impact the README.md documentation during the UA -> GA4 upgrade should be updated

### PR DESCRIPTION
**Pull request by scottqueen-bixal**
- updates readme version path note

fixes #227

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -102,21 +104,15 @@ routesVersioning({
 }, NoMatchFoundCallback));
 
 function NoMatchFoundCallback(req, res) {
-  res.status(404).send("Version not found. See https://analytics.usa.gov/developer");
+  res.status(404).json("Version not found. Visit https://analytics.usa.gov/developer for information on the latest supported version.");
 }
 
 // v1
 function respondV1(req, res) {
```
Comment (ryanwoldatwork) [inline src/app.js:111]: @scottqueen-bixal - i don't have all the context for this loc, but i'm curious why the version would no longer be set and the console log message not shown.    Is `respondV1` function still called anywhere?  If so, I'd still expect it to return a `v1.1` response.     At a glance, I'm not sure if this is the case or not.

Comment (scottqueen-bixal) [reply src/app.js:111]: If a request with `v1.1` is made it does call `respondv1`. 

The `NoMatchFoundCallback` function handles when we don't have a supported version. For example, if I make a request with `v3` in the route path.

Comment (ryanwoldatwork) [reply src/app.js:111]: @scottqueen-bixal - okay, this makes sense.

I was more curious about 

```
 req.params.version = '1.1'
  // TODO - report this message to response
  console.log('v1 is deprecated. Use v2 instead. See https://analytics.usa.gov/developer')
```

in the `respondv1` function.

It looks like the version isn't being set and the deprecation message removed, and I was wondering why in the first comment. 

i guess my question is: if somebody calls `v1.1`, do they still get a v1.1 response?

Comment (scottqueen-bixal) [reply src/app.js]: You were seeing some outdated code on this branch - it's now updated with develop.

We don't log the response there any more. see https://github.com/18F/analytics-reporter-api/blob/develop/src/app.js#L82-L117

Short answer is yes, they absolutely get a response on `v1.1` requests.

**Pull request merged**